### PR TITLE
2150 2152 Enhance metrics summary report

### LINKED ISSUE #2150: Add sample count in metric_summary report
**Is your feature request related to a problem? Please describe.**
Need to add the sample count for every class in the `metric_summary` report, and enhance the doc-string of average computation progress.
Feature request from internal research team.

### PR DESCRIPTION
**Pull request by Nic-Ma**
Fixes #2150 #2152 .

### Description
This PR：
1. added `notnans` count to the metrics summary report。
2. enhanced the doc-strings.
3. add support to set percent value.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/handlers/utils.py [hunk 1]
```diff
@@ -140,15 +140,23 @@ def write_metrics_reports(
...
         summary_ops: expected computation operations to generate the summary report.
-            it can be: None, "*" or list of strings.
-            None - don't generate summary report for every expected metric_details
+            it can be: None, "*" or list of strings, default to None.
+            None - don't generate summary report for every expected metric_details.
             "*" - generate summary report for every metric_details with all the supported operations.
             list of strings - generate summary report for every metric_details with specified operations, they
-            should be within this list: [`mean`, `median`, `max`, `min`, `90percent`, `std`].
-            default to None.
+            should be within list: ["mean", "median", "max", "min", "<int>percent", "std", "notnans"].
```
Comment (wyli) [inline monai/handlers/utils.py:150]: I think `percent` -> `percentile`

**Review by wyli — APPROVED** [review]
thanks! we should replace `percent` with `percentile` in this PR

File: monai/handlers/utils.py (hunk 1 above)
Comment (Nic-Ma) [reply monai/handlers/utils.py]: sure, updated.

**Pull request merged**